recipes/urduhack/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "urduhack" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9170ec2e20a6fd2f0620a6d24d61c4233b2b4afe5c1b7e4d4326ef8f04cc6cb4
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - bsdiff4 = bsdiff4.cli:main_bsdiff4
+    - bspatch4 = bsdiff4.cli:main_bspatch4
+
+requirements:
+  host:
+    - pip
+    - python
+    - regex
+  run:
+    - python
+
```

Comment:
@muhammadfahid51 If your package uses `regex` at runtime, you need to list it in the `run`-section.